src/SecurityInsights/examples/Get-AzSentinelAutomationRule.md

```diff
@@ -0,0 +1,30 @@
+### Example 1: List all Automation Rules
+```powershell
+PS C:\> Get-AzSentinelAutomationRule -ResourceGroupName "myResourceGroupName" -workspaceName "myWorkspaceName"
```

Comment:
```suggestion
Get-AzSentinelAutomationRule -ResourceGroupName "myResourceGroupName" -workspaceName "myWorkspaceName"
```

Could you please remove PS C:\> from your examples to avoid breaking user copy-paste-use experience?

Reply:
Fixed in lastest commit

Reply:
Thank you. To apply these changes in docs/*.md, please run ./build-module.ps1 again and commit your changes again

Reply:
Done